oak_functions/README.md

```diff
@@ -5,17 +5,34 @@ computing platform that allows developing stateless applications in a privacy
 preserving way. Oak Functions leverages TEEs and remote attestation, Wasm
 sandboxing, and allows exposing metrics in a Differentially Private way.
 
+At its core, the Oak Functions loader is a trusted runtime compiled to a server
+binary that, for each incoming client gRPC request, executes an untrusted
+workload that operates on the request payload, and produces a response which is
+sent back to the same client. The trusted runtime ensures that the untrusted
+workload may not violate the confidentiality of the client request data,
+preventing observers from learning anything about the request. From the client
+point of view, the server provides cryptographic evidence (backed by an
+hardware-based Trusted Execution Environment) of its own identity as part of a
+remote attestation protocol, which convinces the client that it is in fact a
+legitimate version of such a runtime.
+
 The main building blocks used in Oak Functions are:
 
 - A Trusted Execution Environment (TEE)
-  - This protects the data from the host, other enclaves and processes, and even
-    some HW attacks.
-  - TEEs can be VM-based (e.g., AMD SEV) or HW-backed (e.g., Intel SGX). Either
-    variant can be used for hosting an Oak Functions application.
-- Sandboxing the workload, currently using WebAssembly
+  - protects the confidentiality and integrity of data and code from the host,
```

Comment:
```suggestion
  - to protect the confidentiality and integrity of data and code from the host,
```
nit: I think it would read better.

Reply:
Most of the other points use third person, so I kept it here too for now, but I agree it should be made more uniform.